handler/asset_handler.go

```diff
@@ -5,11 +5,164 @@ import (
 	"fmt"
 	"net/http"
 
+	"github.com/google/uuid"
+	"github.com/gorilla/mux"
 	"github.com/vkhichar/asset-management/contract"
 	"github.com/vkhichar/asset-management/customerrors"
 	"github.com/vkhichar/asset-management/service"
 )
 
+func DeleteAssetHandler(asset service.AssetService) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		vars := mux.Vars(r)
+		Id, errParse := uuid.Parse(vars["Id"])
+		if errParse != nil {
+			fmt.Println("handler: Invalid UUID")
```

Comment:
remove fmt , don't use fmt.Println to log  , use fmt.Printf with proper request params so that its easier to debug